### DISCUSSION TITLE
音乐播放器修改了歌单id但是不生效

### DISCUSSION THREAD

**Discussion by Hurtblack** [tangly1024/NotionNext#2279] [Q&A]
好像是默认的歌单，怎么改都改不生效
![image](https://github.com/tangly1024/NotionNext/assets/74805072/30b3cb0f-b306-45b0-b7af-53d249b17ff2)

网址：www.ltyzqhh.top

**Comment by erduoya77** [comment]
一样，4.6.1版本
